source/Camera/Advanced/CaptureHDRPrintNormals/CaptureHDRPrintNormals.cpp

```diff
@@ -35,8 +37,11 @@ int main()
     {
         Zivid::Application zivid;
 
-        std::cout << "Connecting to camera" << std::endl;
-        auto camera = zivid.connectCamera();
+        // The fileCamera file is in Zivid Sample Data. See instructions in README.md
+        const auto fileCamera = std::string(ZIVID_SAMPLE_DATA_DIR) + "/FileCameraZividOne.zfc";
+
+        std::cout << "Creating virtual camera using file: " << fileCamera << std::endl;
+        auto camera = zivid.createFileCamera(fileCamera);
```

Comment:
Huh? Why don't you want to use a camera?

Reply:
Mistake :(